### PR TITLE
CLOUDSTACK-9865: improper os types for xenserver 6.5

### PR DESCRIPTION
**Pull request by pdion891**
Windows 2016 is not listed as Template in XenServer 6.5.
New Debian 8 was not defined for XenServer 6.5

### CODE REVIEW

**Comment by pdion891** [comment]
VM creation tested for Master branch + XenServer6.5:  Windows 2016, Centos7, CoreOS, Windows 2012, Debian8, Ubuntu 16.04

**Comment by cloudmonger** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 813
 Hypervisor xenserver
 NetworkType Advanced
 Passed=109
 Failed=4
 Skipped=12

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/r2si930m8xxzavs/AAAzNrnoF1fC3auFrvsKo_8-a?dl=0

**Failed tests:**
* test_routers_network_ops.py

 * test_01_isolate_network_FW_PF_default_routes_egress_true Failing since 9 runs

 * test_02_isolate_network_FW_PF_default_routes_egress_false Failing since 9 runs

 * test_01_RVR_Network_FW_PF_SSH_default_routes_egress_true Failing since 9 runs

 * test_02_RVR_Network_FW_PF_SSH_default_routes_egress_false Failing since 9 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_01_verify_libvirt
test_02_verify_libvirt_after_restart
test_03_verify_libvirt_attach_disk
test_04_verify_guest_lspci
test_05_change_vm_ostype_restart
test_06_verify_guest_lspci_again
test_static_role_account_acls
test_11_ss_nfs_version_on_ssvm
test_nested_virtualization_vmware
test_3d_gpu_support
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_vm_snapshots.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_loadbalance.py
test_routers.py
test_reset_vm_on_reboot.py
test_deploy_vms_with_varied_deploymentplanners.py
test_network.py
test_router_dns.py
test_non_contigiousvlan.py
test_login.py
test_deploy_vm_iso.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_metrics_api.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_volumes.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_vm_life_cycle.py
test_disk_offerings.py

File: setup/db/db/schema-4920to41000.sql [hunk 1]
```diff
@@ -48,7 +48,7 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervi
 INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(), 'VMware', '6.0', 'windows9_64Guest', 258, now(), 0);
 INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(), 'KVM', 'default', 'Windows 10', 258, now(), 0);
 
-INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(), 'Xenserver', '6.5.0', 'Windows Server 2016 (64-bit)', 259, now(), 0);
```
Comment (rhtyd) [inline setup/db/db/schema-4920to41000.sql]: Can this be moved to 4.10->4.11 upgrade path, thanks

**Comment by rhtyd** [comment]
@pdion891 can you fix the conflicts and move sql changes to 4.10->4.11 upgrade path sql, thanks

**Comment by rhtyd** [comment]
@pdion891 ping, please fix the conflicts

**Comment by rhtyd** [comment]
@pdion891 please fix conflicts and move changes to 4.10->4.11 sql

**Comment by rhtyd** [comment]
@pdion891 can you fix the conflicts? /cc @syed @swill

**Comment by rhtyd** [comment]
I've removed this from 4.11 milestone due to lack of engagement from the author @pdion891

**Comment by pdion891** [comment]
Closing this PR,  because of the merge conflit and also recent changes on XenServer.

Sorry @rhtyd  for my lack of follow up, but thanks a lot for yours !  Cheers,

**Pull request closed without merging**